.github/workflows/publish-cdk-command-manually.yml

```diff
@@ -97,7 +97,7 @@ jobs:
         with:
           repository: ${{ github.event.inputs.repo }}
           ref: ${{ github.event.inputs.gitref }}
-          token: ${{ secrets.GH_PAT_MAINTENANCE_OSS }}
+          token: ${{ secrets.GH_PAT_MAINTENANCE_OSS }} # This token is what allows us to commit directly to master
       - name: "Publish Python CDK: bump Poetry package version"
         id: bumpversion
         run: |
```

Comment:
Oh I love this, now I know where we have the logic that pushes the version into the Changelog.